solutions/remotemonitoring/single-vm/setup.sh

```diff
@@ -66,6 +66,38 @@ config_for_azure_china() {
 
 config_for_azure_china $HOST_NAME $5
 
+# ========================================================================
+# Configure SSH to not use weak HostKeys, algorithms, ciphers and MAC algorithms.
+# Comment out the option if exists or ignore it.
+switch_off() {
+    local key=$1
+    local value=$2
+    local config_path=$3
+    sed -i "s~#*$key\s*$value~#$key $value~g" $config_path
+}
+
+# Change existing option if found or append specified key value pair.
+switch_on() {
+    local key=$1
+    local value=$2
+    local config_path=$3
+    grep -q "$key" $config_path && sed -i -e "s/$key.*/$key $value/g" $config_path || sed -i -e "\$a$key $value" $config_path
+}
+
+config_ssh() {
+    local config_path="${1:-/etc/ssh/sshd_config}"
+    switch_off 'HostKey' '/etc/ssh/ssh_host_dsa_key' $config_path
+    switch_off 'HostKey' '/etc/ssh/ssh_host_ecdsa_key' $config_path
+    # TODO: Change to no to disable tunnelled clear text passwords
+    # switch_on 'PasswordAuthentication' 'no' $config_path
```

Comment:
is this a comment left in on purpose or code you want?

Reply:
this is the code we may want to disable password authentication after GA. But now we are not going to to  because it will prevent user to login by password if applied. When we change to use ssh key to login, we should be able to uncomment this code. Is it good to put here for GA?